challenges/objects-arrays.js

```diff
@@ -58,12 +93,26 @@ The resulting contact information should have a space between the first name and
 Name email@example.com
 
 Log the result of your new array. */
-const contactInfo = [];
+const contactInfo = [{"first_name":"Cynde","email":"ctorry0@macromedia.com"},
+{"first_name":"Saundra","email":"swhal1@state.gov"},
+{"first_name":"Lambert","email":"lparham2@techcrunch.com"},
+{"first_name":"Modestine","email":"mdolder3@symantec.com"},
+{"first_name":"Chick","email":"camorts4@google.com.au"},
+{"first_name":"Jakob","email":"jharken5@spiegel.de"},
+{"first_name":"Robbi","email":"rbrister6@redcross.org"},
+{"first_name":"Colline","email":"cbrosh7@alibaba.com"},
+{"first_name":"Michail","email":"mrome8@shinystat.com"},
+{"first_name":"Hube","email":"hlethbrig9@foxnews.com"}];
 console.log(contactInfo);
 
 
 /* Request 3: Find out how many universities have the string "Uni" included in their name. Create a new array called uni that contains them all. Log the result. */
-const uni = [];
+const uni = [
+{"university":"International Medical & Technological University"},
+{"university":"Salem University"},
+{"university":"Coastal Carolina University"},
+{"university":"Universidad Católica de Ávila"},
+{"university":"Universitat Rovira I Virgili Tarragona"}];
 console.log(uni);
```

Comment:
Use JavaScript to create the new arrays in request 1, 2, and 3. You could use a `for` loop to iterate through your original array and create the new array.